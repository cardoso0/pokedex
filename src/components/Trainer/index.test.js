import { render } from "@testing-library/react"
import "@testing-library/jest-dom"
import renderer from "react-test-renderer"

import { Trainer } from "."

const renderTrainer =
  <Trainer />

describe("Trainer Component", () => {
  it ("component rendering | snapshot", () => {
    const tree = renderer
    .create(renderTrainer)
    .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it ("component rendering", () => {
    expect(render(renderTrainer)).toBeTruthy
  })
})