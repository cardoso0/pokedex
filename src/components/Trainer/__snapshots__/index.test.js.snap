// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trainer Component component rendering | snapshot 1`] = `
<div
  className="sc-bczRLJ jhlpqo"
>
  <h1 />
  <figure>
    <img />
  </figure>
  <div
    className="sc-gsnTZi cKbZtm"
  >
    <table>
      <tbody>
        <tr>
          <th
            scope="row"
          />
          <td />
        </tr>
        <tr>
          <th
            scope="row"
          />
          <td />
        </tr>
        <tr>
          <th
            scope="row"
          />
          <td />
        </tr>
      </tbody>
    </table>
    <p />
    <p />
  </div>
</div>
`;
